openshift-container-platform-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -89,6 +89,25 @@
         benchmarks to remediate high, and moderate risk flaws within a
         customer defined period after a flaws discovery.
         */'
+
+- control_key: SI-2(4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: complete
+  narrative:
+    - key: a
+      text: |
+        Withdrawn
+
+- control_key: SI-2(5)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - key: a
+      text: |
+        SI-2(5) is an organizational control outside the scope of OpenShift configuration.
```

Comment:
Same general feedback as SI-2 (4), in that:
- Update the control_key to have spaces (``SI-2(5)`` --> ``SI-2 (5)``)
- No need for ``key: a``. An example of how the ``narrative`` should be setup (when there are not subelements) can be borrowed from AC-3 (2):

https://github.com/ComplianceAsCode/redhat/blob/master/openshift-container-platform-4/policies/AC-Access_Control/component.yaml#L288#L297

`````
- control_key: AC-3 (2)
  standard_key: NIST-800-53
  covered_by: []
  implementation_status: planned
  narrative:
    - text: |-
        A control response for AC-3 (2) is planned. Engineering
        progress can be tracked via:
        https://issues.redhat.com/browse/CMP-88
`````

As for a response to SI-2 (5): There are configuration options OpenShift should enforce relating to auto patching. For example, if a base container was updated (eg RHEL UBI), should we ensure any apps using that UBI image get re-deployed? How is that configured? Same with any layered container image (eg Apache, EAP....).